src/global.ts

```diff
@@ -0,0 +1,31 @@
+import type { Unimport } from './context'
+import type { Import, InstallGlobalOptions } from './types'
+
+export async function installGlobalAutoImports (
+  imports: Import[] | Unimport,
+  options: InstallGlobalOptions = {}
+) {
+  const {
+    globalObject = globalThis,
+    overrides = false
+  } = options
+
+  imports = Array.isArray(imports)
+    ? imports
+    : await imports.getImports()
+
+  await Promise.all(
+    imports.map(async (i) => {
+      if (i.disabled) {
+        return
+      }
+      const as = i.as || i.name
+      if (overrides || !(as in globalObject)) {
+        const module = await import(i.from)
+        globalObject[as] = module[i.name]
```

Comment:
(do we need to add a sanity check and avoid assignment if `i.name` not exists in module?

Reply:
I think that makes sense. But if we do so, should we warn that the name does not exists, or ignore silently?